lnrpc/walletrpc/walletkit_server.go

```diff
@@ -993,6 +1002,11 @@ func validateBumpFeeRequest(in *BumpFeeRequest) (
 		immediate = in.Immediate
 	}
 
+	if in.DeadlineDelta != 0 && in.Budget == 0 {
```

Comment:
hmm think this would prevent user from updating a single param `deadline-delta` , but i guess they can always et the current budget from `PendingSweeps` if they want to update the deadline only.

Reply:
yeah I think it's better that way.